app/emulator/include/emulator.hpp

```diff
@@ -27,18 +27,39 @@
 #define EMULATOR_GEAR_RATIO_3 15  // 1.5:1
 #define EMULATOR_GEAR_RATIO_4 10  // 1:1
 #define EMULATOR_GEAR_RATIO_5 5   // 0.5:1
+#define PINDLE_PARKING 112  // Gear parking, dec 112
+#define PINDLE_NEUTRAL 110  // Gear parking, dec 112
+#define PINDLE_DRIVE 100  // Gear parking, dec 112
+#define PINDLE_REVERSE 114  // Gear parking, dec 112
 
 
 typedef struct EmulatorData {
   size_t throttle_set_value = 0;
   size_t throttle;
-  size_t gear_set_value = 112;
-  size_t start_set_value = 0;
+  size_t gear_set_value = PINDLE_PARKING;
+  size_t start_set_value = false;
   size_t gear;
   size_t rpm;
   size_t speed;
+  bool activate_engine = false;
+  bool gear_neutral = false;
+  bool gear_drive = false;
```

Comment:
why not just use one char variable that is char pindle_value that can take the values P, D, R, N instead of 4 bool? :)

Reply:
Because some of them will be used in conjunction, and since I use them as flags in the logic and they also will be used by the emulator, it's easier to have them as separate flags.